packages/aws-rfdk/lib/deadline/test/worker-configuration.test.ts

```diff
@@ -71,14 +70,35 @@ describe('Test WorkerInstanceConfiguration for Linux', () => {
     });
     const userData = stack.resolve(instance.userData.render());
 
-    // THEN
+    // // THEN
+    expect(userData).toStrictEqual({
+      'Fn::Join': [
+        '',
+        [
+          '#!/bin/bash\nmkdir -p $(dirname \'/tmp/',
+          ...linuxConfigureWorkerScriptBoilerplate(`\' \'\' \'\' \'\' \'${Version.MINIMUM_SUPPORTED_DEADLINE_VERSION.toString()}\' 56032 /tmp/`),
```

Comment:
Would it be better to use `DEFAULT_LISTENER_PORT ` instead of 56032?
It could make code more understandable.